expression/builtin_math_vec.go

```diff
@@ -40,3 +41,28 @@ func (b *builtinLog10Sig) vecEvalReal(input *chunk.Chunk, result *chunk.Column)
 func (b *builtinLog10Sig) vectorized() bool {
 	return true
 }
+
+func (b *builtinAbsDecSig) vecEvalDecimal(input *chunk.Chunk, result *chunk.Column) error {
+	if err := b.args[0].VecEvalDecimal(b.ctx, input, result); err != nil {
+		return err
+	}
+	zero := new(types.MyDecimal)
+	buf := new(types.MyDecimal)
+	d64s := result.Decimals()
+	for i := 0; i < len(d64s); i++ {
+		if result.IsNull(i) {
```

Comment:
To calculate `result.IsNull()`, it's better to use `result.MergeNulls()`.

Reply:
it seems `MergeNulls` is to merge the null for two column and there is only one......I feel confused about how to use it here...

Reply:
Do we need to `set d64s[i] null` if `result.IsNull`?